base/util_testing.go

```diff
@@ -891,3 +891,18 @@ func AssertTimestampGreaterThan(t *testing.T, e1, e2 int64, msgAndArgs ...interf
 	}
 	return assert.Greater(t, e1, e2, msgAndArgs...)
 }
+
+func GetVbucketForKey(bucket Bucket, key string) (vbNo uint32, err error) {
```

Comment:
I'm inclined to make this a function in rest package and private so that if we are not longer using it, we can remove it.

OTOH, the likelihood we'll use it again is moderate.

Reply:
This is a useful utility function, IMO - I've had many cases where it's useful to be able to identify vbucket for a key.